src/components/pages/gallery/SelectedFileOptions/DeduplicateOptions.tsx

```diff
@@ -76,15 +76,13 @@ export default function DeduplicateOptions({
                     width: '1em',
                     height: '1em',
                 }}
-                value={
-                    deduplicateContext.clubSameFileHashesOnly ? 'true' : 'false'
-                }
+                value={deduplicateContext.clubBySameSize ? 'true' : 'false'}
                 onChange={() => {
-                    deduplicateContext.setClubSameFileHashesOnly(
-                        !deduplicateContext.clubSameFileHashesOnly
+                    deduplicateContext.setClubBySameSize(
```

Comment:
why do we need this?

Reply:
I replaced state for hashes by state for same size since now hashes is default. Should we instead keep the hashes state?